### PR TITLE
Mongo migration

### PR DESCRIPTION
**Pull request by koosoku**
Hi friends, migrating to Mongo, all tests are updated as well as the readme :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "cassandra-driver": "^3.3.0",
     "dotenv": "^5.0.1",
     "express": "^4.16.2",
-    "jsonwebtoken": "^8.1.1"
+    "jsonwebtoken": "^8.1.1",
+    "mongodb": "^3.0.4"
```
Comment (silverAndroid) [inline package.json]: if cassandra has been completely removed, remove `cassandra-driver` as a dependency

File: README.md [hunk 2]
```diff
@@ -3,3 +3,9 @@ To setup cassandra with docker:
 docker pull cassandra
```
Comment (singhprincejeet) [inline README.md]: why didn't we remove cassandra reference from here

File: package.json (hunk 1 above)
Comment (koosoku) [reply package.json]: yup!  will do :)

File: README.md (hunk 2 above)
Comment (koosoku) [reply README.md]: my bad :)

File: src/models/users_model.js [hunk 3]
```diff
@@ -1,168 +1,81 @@
...
-                    console.log(`Drop table error: ${err}`);
-
-                    return next(err);
-                }
-                console.log("Table dropped");
-                next();
-            });
-        }
-    ], callback);
+const dropRecipeTable = (client, collection, next) => {
```
Comment (singhprincejeet) [inline src/models/users_model.js]: I guess u mean user table

File: src/routes/users_routes.js [hunk 4]
```diff
@@ -2,9 +2,9 @@ const app = require("express").Router();
 const usersController = require("../controllers/users_controller");
 const {env} = require("../../config");
 
-if (env !== "testing") {
-    usersController.setUp();
-}
+// if (env !== "testing") {
```
Comment (singhprincejeet) [inline src/routes/users_routes.js]: maybe not push commented code to master

**Pull request merged**